transaction-metrics-tracker/src/lib.rs

```diff
@@ -0,0 +1,54 @@
+use {
```

Comment:
can we add tests for this module? especially around `get_signature_from_packet` with truncated inputs

Reply:
Done.